ckanext/canada/controller.py

```diff
@@ -262,6 +266,40 @@ def fgpv_vpgf(self, pkg_id):
         return render('fgpv_vpgf/index.html', extra_vars={
             'pkg_id': pkg_id,
         })
+    def data_dictionary(self, pd_file):
+        names = pd_file.split('.')
+        if not (len(names)==2 and names[1]=='csv'):
+            return None
+        chromo = h.recombinant_get_chromo(names[0])
+        if not chromo:
+            return None
+
+        current_lang= get_lang()
+        set_lang('fr')
+        csv_dict = [(u'Field Name / Nom de la zone',
+            u'English Description / Description en anglais',
+            u'French description / Description en français')]
+        csv_dict += [ (f['datastore_id'], f['label'], _(f['label']))
+                for f in chromo['fields'] ]
+        set_lang(current_lang)
+
+        csv_dict.append(('owner_org', 'owner organization',
+                _('owner organization') ))
+        csv_dict.append(('owner_org_title', 'owner organization title',
+                _('owner organization title') ))
+
+        blob = StringIO()
```

Comment:
There's no need for StringIO. We can .write straight to the pylons response object.